variables.tf

```
@@ -106,6 +106,12 @@ variable "instance_parameters" {
   description = "List of DB instance parameters to apply"
 }
 
+variable "db_cluster_instance_class" {
+  type        = string
+  default     = ""
```

Lets default this to `null`
```suggestion
  default     = "null`
```